packages/runtime/src/components/builtin/Slot/register.ts

```diff
@@ -0,0 +1,20 @@
+import { lazy } from 'react';
+import { ReactRuntime } from '../../../react'
+import { MakeswiftComponentType } from '../constants'
+import { Checkbox } from '@makeswift/controls'
+import { Slot } from '../../../controls';
```

Comment:
If you have conflicts with this name when renaming `SlotRoot` to `Slot`, you can alias this import like so

```suggestion
import { Slot as SlotControl } from '../../../controls';
```